### PR TITLE
added restartsec and startlimitinterval configurations

### PR DESCRIPTION
**Pull request by oguzhaninan**

### CODE REVIEW

File: systemd/vmware_exporter.service [hunk 1]
```diff
@@ -1,13 +1,16 @@
 [Unit]
 Description=Prometheus VMWare Exporter
 After=network.target
+StartLimitInterval=0
```
Comment (pryorda) [inline systemd/vmware_exporter.service]: Are you wanting StartLimitIntervalSec?

Comment (pryorda) [reply systemd/vmware_exporter.service]: Are we sure we want this zero?

File: systemd/vmware_exporter.service [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 [Service]
 User=prometheus
 Group=prometheus
 WorkingDirectory=/opt/prometheus/vmware_exporter/
 ExecStart=/opt/prometheus/vmware_exporter/vmware_exporter.py
 Type=simple
+Restart=always
+RestartSec=1
```
Comment (pryorda) [inline systemd/vmware_exporter.service]: Seems quick what about 5 or even 10?

**Comment by oguzhaninan** [comment]
By default, when you configure Restart=always as we did, systemd gives up restarting your service if it fails to start more than 5 times within a 10 seconds interval. Forever.
There are two [Unit] configuration options responsible for this:
StartLimitBurst=5
StartLimitInterval=10

The simple fix that always works is to set StartLimitInterval=0. This way, systemd will attempt to restart your service forever.
It’s a good idea to set RestartSec to at least 1 second though, to avoid putting too much stress on your server when things start going wrong.
[Source](https://medium.com/@benmorel/creating-a-linux-service-with-systemd-611b5c8b91d6)

**Comment by pryorda** [comment]
> By default, when you configure Restart=always as we did, systemd gives up restarting your service if it fails to start more than 5 times within a 10 seconds interval. Forever.
> There are two [Unit] configuration options responsible for this:
> StartLimitBurst=5
> StartLimitInterval=10
> 
> The simple fix that always works is to set StartLimitInterval=0. This way, systemd will attempt to restart your service forever.
> It’s a good idea to set RestartSec to at least 1 second though, to avoid putting too much stress on your server when things start going wrong.
> [Source](https://medium.com/@benmorel/creating-a-linux-service-with-systemd-611b5c8b91d6)

The article shows different from what  you are proposing. 

StartLimitBurst=5
StartLimitIntervalSec=10 

vs 

> StartLimitBurst=5
> StartLimitInterval=10

**Comment by pryorda** [comment]
@oguzhaninan Did you close this because of the questions or because you decided it was no longer relevant?

**Comment by oguzhaninan** [comment]
@pryorda You ask simple questions and I don't have time. Sorry.

**Comment by pryorda** [comment]
👍

**Pull request closed without merging**